src/esm_runscripts/namelists.py

```diff
@@ -491,7 +492,18 @@ def apply_iceberg_calving(config):
                     ib_num_old = 0
 
                 print(" * iceberg_dir = ", config["fesom"].get("iceberg_dir"))
-                ib_num_new = sum(1 for line in open(config["fesom"].get("iceberg_dir") + "/LON.dat"))
+                iceberg_dir = config["fesom"].get("iceberg_dir", None)
+                if os.path.isfile(iceberg_dir + "/LON.dat"):
+                    ib_num_new = sum(1 for line in open(config["fesom"].get("iceberg_dir") + "/LON.dat"))
+                elif config["general"]["check"]:
```

Comment:
I am not sure I like this, since the `check` mode might be on also for other reasons. Can we make it so that `esm-tests` gets its own flag for `esm-runscripts`?

Reply:
The above is also an outdated comment from @denizural `enum` suggestion.

Reply:
Yeah, this is horrible... I'll find a better way to fix this.